src/themes/lv_theme.h

```diff
@@ -93,6 +106,13 @@ lv_color_t lv_theme_get_color_primary(lv_obj_t * obj);
  */
 lv_color_t lv_theme_get_color_secondary(lv_obj_t * obj);
 
+
+/**
+ * Destroy a theme
+ * @param   theme the theme to destroy
+ */
+void lv_theme_destroy(lv_theme_t * theme);
```

Comment:
```suggestion
void lv_theme_delete(lv_theme_t * theme);
```

I know we use "destroy" on a few places but "delete" is more common in LVGL.

Reply:
I'm sorry, I took a break from doing wayland to this. Wayland api is full of "destroy". I fixed it on 9754c99